src/generators/web.cr

```diff
@@ -89,171 +82,80 @@ class LuckyCli::Generators::Web
     if default_directory?
       project_dir
     else
-      [full_project_directory.chomp('/'), project_dir].join('/')
+      File.join(full_project_directory.chomp('/'), project_dir)
```

Comment:
I'm not sure how this looks on Windows, by we may need to see if there's some sort of `Crystal::System::FileSeparator` or whatever. I'm assuming on Windows we'd need to `chomp('\')`...

Reply:
While you _can_ use `File` methods for dealing with `Path`s, because they are just wrappers around `Path`, I'd recommend to just use `Path` directly instead. It makes it more clear what you're doing (e.g. path manipulation), and use `File` for actual filesystem operations (e.g. read/write a file).

You don't actually need `#chomp` here:
```suggestion
      File.join(full_project_directory, project_dir)
```

But this is my suggestion (which is the same as `File#join`, just prefer using `Path`):
```suggestion
      Path.new(full_project_directory).join(project_dir).to_s
```

However, my above suggestion would be better suited in a separate PR to change _all other_ places using `File` instead of `Path` (I might add this to my lucky_template work).

Reply:
Yeah, I never thought about using `Path` until I started seeing your code. I think that makes more sense. Maybe I'll just merge this PR in as-is, then in another one we can go through it all and really start cleaning stuff up.